python-sdk/nuscenes/tests/test_predict_helper.py

```diff
@@ -0,0 +1,503 @@
+# nuScenes dev-kit.
+# Code written by Freddy Boulton, 2020.
+
+import unittest
+from typing import Dict, List, Any
+from nuscenes.predict import PredictHelper, convert_global_coords_to_local
+import numpy as np
+
+class MockNuScenes:
```

Comment:
Propose to use inheritance here to make it cleaner. `MockNuscenes` simply is a NuScenes subclass that overwrites some methods.